app/javascript/controllers/signup_controller.js

```diff
@@ -0,0 +1,20 @@
+import { Controller } from "@hotwired/stimulus";
+
+// Connects to data-controller="signup"
+export default class extends Controller {
+  static targets = ["mentorForm", "menteeForm"];
+
+  connect() {
+    this.mentorFormTarget.hidden = true;
+  }
+
+  showForm({ params: { content } }) {
```

Comment:
Love that you added the ability to switch between both forms depending on role, will make it a lot easier for routing down the road.

Reply:
Thank you bro!